runtime/src/assets/stylesheets/shared/_custom-variables.less

```diff
@@ -0,0 +1,16 @@
+@import (reference) './../../../../node_modules/patternfly/node_modules/bootstrap/less/variables';
+@import (reference) './../../../../node_modules/patternfly/src/less/variables';
+
+//custom variables
+@width100: 100%;
```

Comment:
remove and replace with `width: 100%;` across files, as this will never not be `width: 100%`, thus not qualifying as a variable under the code guidelines.